packages/js/storybook/README.md

```diff
@@ -0,0 +1,24 @@
+# Grow Team Storybook
+
+Storybook dependencies and basic scripts
+
+
+## Prerequisites
+
+- You will the StoryBook config files as if you installed it in your ouw repository.
+- You will need to configure GH Pages s if you would do it in your own repository.
+- You will need to add the scripts in your repo calling the storybook binaries described in the usage section. 
```

Comment:
:construction: 
I'd remove repeated, "You will". Also, AFAIK  "Prerequisites" means that something needs to be done as a prior condition.

Also, I'd remove the last point, as it is more of a "usage" than a prerequisite.
```suggestion
- Create StoryBook config files as if you installed it in your own repository.
- Configure GH Pages as if you would do it in your own repository.
```